.github/workflows/dapr-bot-schedule.yml

```diff
@@ -0,0 +1,47 @@
+# ------------------------------------------------------------
```

Comment:
@daixiang0 Please change the license header ...

Reply:
done.